cmd/lncli/commands.go

```diff
@@ -422,6 +422,13 @@ var openChannelCommand = cli.Command{
 				"sat/byte that should be used when crafting " +
 				"the transaction",
 		},
+		cli.BoolFlag{
+			Name: "private",
+			Usage: "make the channel private, such that it won't " +
+				"be announced to the greater network, and " +
+				"only the two channel endpoints can use it " +
```

Comment:
This isn't _quite_ accurate, as we'll be using the payreqs to give routing hints in the future. Instead, we can say something like: "other nodes won't be able to route to this channel unassisted".